CHANGELOG.md

```diff
@@ -2,6 +2,20 @@
 
 ## 0.19
 
+* Workflow options may now contain a choice of backend specific for that workflow:
+```
+{
+  "backend": "JES"
+}
+```
+* To support the above change, the configuration file has changed how backends are specified. You must replace `backend.backend` with `backend.defaultBackend`. 
+In addition the option `backend.backendsAllowed` must be specified, for example:
+```
+backend {
+  defaultBackend = "local"
+  backendsAllowed = [ "local", "JES", "SGE" ]
```

Comment:
Just wondering since Ruchi had this issue yesterday, but I assume that `defaultBackend` must be in the list of `backendsAllowed` ? If yes is it worth mentioning ?